src/runtime/auto-prompt-manager.js

```diff
@@ -531,8 +532,8 @@ export class AutoPromptManager {
       );
```

Comment:
Comment on line 527: `return TYPE_REWARDED_SURVEY;`
Should we convert it to an `Intervention` instance? 
(I think it would've caught by tests. Do you know why it wasn't?)

Not sure why I can't comment on the line :'(

Reply:
expanded the field `interventionDisplayed_` to include the configurationId. updated the survey tests to also assert equality on this field.

Reply:
I like how they are renamed! Thanks